Add override to toggle agent-configuration job (#5569)
kyma-project/kyma: resources/compass/README.md

| Parameter | Description | Values | Default |
| --- | --- | --- | --- |
| **database.useEmbedded** | Specifies whether `postgresql` chart should be installed. | true/false | `true` |

> | **agentPreconfiguration** | Specifies whether `compass-agent-configuration` job will be triggered to provide static configuration for runtime agent. | true/false | `true` |

Reviewer: ```suggestion
| **agentPreconfiguration** | Specifies whether the `compass-agent-configuration` job is triggered to provide static configuration for the Runtime Agent. | true/false | `true` |
```
